README.md

```diff
@@ -2,7 +2,33 @@
 
 ## Purpose
 
-This team is focused around how the core http(1/2/3) and other related apis fit the needs of the web frameworks (express, fastify, restify, hapi, koa etc). It also gives the maintainers a forum to discuss and collaborate on issues which span the projects.
+The Web-Server Frameworks Team is a place for Node.js framework authors and users to collaborate on the
+future direction and feature set of the frameworks and related Node.js core APIs.  The main goal of the
+team is to improve collaboration across the ecosystem and foster a better dialog between all parties involved.
+
+### What are Web-Server Frameworks
+
+A web-server framework is a tool/library/package which offers a higher level abstraction for how users interact
+with over the wire protocols (ex. http1/2/3 or WebSockets).  In the Node.js ecosystem there are a few notable examples:
+
+- [Express](https://github.com/expressjs/express/)
+- [Fastify](https://github.com/fastify/fastify)
+- [hapi](https://github.com/hapijs/hapi)
```

Comment:
Hapi?

Reply:
What about Koa?

Reply:
This was not intended as a comprehensive list.  Just a list to get the context.  Maybe I should just remove it as this is the second question about why something is not on the list.